sources/packages/api/src-sql/ProgramYear/Add-col-start-end-dates copy.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+  sims.program_years
+ADD
+  COLUMN IF NOT EXISTS start_date DATE NOT NULL DEFAULT NOW();
+
+ALTER TABLE
+  sims.program_years
+ADD
+  COLUMN IF NOT EXISTS end_date DATE NOT NULL DEFAULT NOW();
+
+COMMENT ON COLUMN sims.program_years.start_date IS 'Inclusive start date for the program year.';
+
+COMMENT ON COLUMN sims.program_years.end_date IS 'Inclusive end date for the program year.';
```

Comment:
why is the file name suffixed with " copy"?

Reply:
It is wrongly named. I will fix it 👍

Reply:
Done!